.travis.yml

```diff
@@ -1,13 +1,69 @@
-sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: make test
+# Two types of tests:
+#
+# 1. shell tests run using bats
+# 2. build tests also run using bats
+#
+# Shell tests are run unconditionally on Linux (see jobs.include).
+#
+# Build tests can be skipped by starting the commit message with
+# '[skip build]'.
+#
+# Build tests are run on platforms generated from all combinations
+# of keys below.
 language: c
+
+os:
+- linux
+- osx
+
+dist:
+- xenial
+
+# https://docs.travis-ci.com/user/reference/osx/#macos-version
+osx_image:
+- xcode9.4
+- xcode10
+
 env:
-  - PYENV_NATIVE_EXT=
-  - PYENV_NATIVE_EXT=1
+- PYTHON_BUILD_VERSION=3.8-dev
+- PYTHON_BUILD_VERSION=3.7.2
+
+before_install:
+- date +%Y-%m-%dT%H:%M:%S
```

Comment:
What is this for...?

Reply:
When running tests I noticed a few times that the workers would hang for a bit after the "Worker Information" and "Build System Information" printed. `before_install` executes before any of our tests or setup so it could be used in conjunction with the test start time (visible by hovering over the duration in the far right column of any build [here](https://travis-ci.com/chrahunt/pyenv/builds/101372597)) to rule out our tests as being the reason for some builds taking a while.